.github/workflows/python-build-and-test.yml

```diff
@@ -7,7 +7,7 @@ jobs:
 
     runs-on: ubuntu-latest
     env:
-      working-directory: /code/pyquarkchain
+      working-directory: ./code/pyquarkchain
```

Comment:
`/code/pyquarkchain` should be the working directory

Reply:
done